packages/prisma/seed.ts

```diff
@@ -169,6 +171,51 @@ async function createTeamAndAddUsers(
   }
 }
 
+// Generates defined number of random users, with extraFields as needed
+async function generateFakeUsers({
+  count = 1,
+  extraFields,
+}: {
+  count: number;
+  extraFields?: Record<string, unknown>;
```

Comment:
We can add the extact the type for createUserAndEventType and pick so we have some type saftey here 

```suggestion
    extraFields?: Pick<SeedUser, "timeZone" | "role">;
```